pennylane/fourier/qnode_spectrum.py

```diff
@@ -399,30 +399,31 @@ def wrapper(*args, **kwargs):
                     f"The return_type {m.return_type.value} is not supported as it likely does "
                     "not admit a Fourier spectrum."
                 )
-        class_jacs = jac_fn(*args, **kwargs)
+        cjacs = jac_fn(*args, **kwargs)
```

Comment:
was this just changed due to consistency with other parts of the codebase?

Reply:
Yes exactly :)